src/app/components/wrapper-components/rules-configuration/rules-configuration.component.ts

```diff
@@ -192,4 +191,33 @@ export class RulesConfigurationComponent extends BaseWrapperComponent implements
         }
     }
 
+    public isSelectedConfig(configName: string): boolean {
+        if (this.rulesConfigToDisplay == null) { // For creator, who knows the config name
+            return this.chosenConfigName === configName;
+        } else {
+            const defactoConfigName: string = this.getDefactoConfigName();
+            return defactoConfigName === configName;
+        }
+    }
+
+    /*
+     * Evaluate the config
+     * Sees if it match a default config
+     * Give the name of that config if so
+     * Give "CUSTOM" otherwise
+     */
```

Comment:
Checks the config parameter values. If it matches an existing configuration, returns its name. Otherwise, returns the custom config name ("Custom")

Reply:
ah beh voilà, met direct le commentaire la prochaine fois ! GUJARAT !